### PR TITLE
Van 120413 find vanity name for api calls

### PR DESCRIPTION
**Pull request by emily-watt-visier**
Add instructions to README.md to find vanity name to call Visier APis. See [VAN-120413](https://visiercorp.atlassian.net/browse/VAN-120413).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ The Visier Python Connector allows Python developers to query against the Visier
 
 ## Prerequisites
 This connector acts as a bridge between your Python application and Visier's cloud-hosted service infrastructure. To successfully connect to your Visier data, you need:
-* The URL domain name prefix. For example: `https://{vanity-name}.api.visier.io`.
+* The URL domain name prefix. For example: `https://{vanity-name}.api.visier.io`. To find your vanity name:
```
Comment (hareskoug) [inline README.md:7]: Odd that it says 'For example`, when no example is given. Instead, we give a pattern. I'd like to take this opportunity to address this weirdness.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,9 @@ The Visier Python Connector allows Python developers to query against the Visier
...
+  1. In Visier, in the global workspace, click **Settings > Single Sign-On** or **Partner Single Sign-On**.
```
Comment (hareskoug) [inline README.md:8]: Nit: should we separate `Settings` from the next option as `Settings` applies to both options?

File: README.md (hunk 1 above)
Comment (emily-watt-visier) [reply README.md:7]: Such as replacing {vanity-name} with jupiter? We can, but then the user won't know what we mean by "vanity name" in the subsequent instructions. A bigger change like:
> Your Visier vanity name and API URL. For example: `https://jupiter.api.visier.io`. To find your vanity name...

File: README.md [hunk 3]
```diff
@@ -4,7 +4,9 @@ The Visier Python Connector allows Python developers to query against the Visier
 
 ## Prerequisites
 This connector acts as a bridge between your Python application and Visier's cloud-hosted service infrastructure. To successfully connect to your Visier data, you need:
-* The URL domain name prefix. For example: `https://{vanity-name}.api.visier.io`.
+* The URL domain name prefix with the following pattern: `https://{vanity-name}.api.visier.io`. To find your vanity name:
+  1. In Visier, in the global workspace, click **Settings**, then click **Single Sign-On** or **Partner Single Sign-On**.
+  2. Under **Single Sign-On** or **Bypass Users (Optional)**, find your service provider endpoint; for example, `https://jupiter.visier.com/VServer/auth`. In this example, `jupiter` is the vanity name.
```
Comment (emily-watt-visier) [inline README.md:9]: @hareskoug Made your suggested changes above

**Review by hareskoug — APPROVED** [review]

**Pull request merged**